### ISSUE TITLE
Cognitive aura

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6475]
**Preferred term label:**

Cognitive aura

**Synonyms**

**Definition (free text, please give PubMed ID)**
A cognitive aura is a purely subjective cognitive ictal symptom.
PMID:30782582

**Parent term (use hpo.jax.org/app)**
two parents -- 
Psychic auras HP:0011162  (Note should be made singular)
Focal cognitive seizure

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

Suggestion of @DLewisSmith

**Comment by pnrobinson** [comment]
Children of new term:


HP:0032692 | Focal cognitive seizure with forced thinking |   |   |  
HP:0032703 | Focal cognitive seizure with hallucination |   |   |  
HP:0032695 | Focal cognitive seizure with illusion |   |   |  
HP:0032697 | Focal cognitive seizure with deja vu/jamais vu |   |   |

**Comment by pnrobinson** [comment]
@DLewisSmith
Where does menomic aura fit in?

https://hpo.jax.org/app/browse/term/HP:0012004

**Comment by pnrobinson** [comment]
We need to create a new term for Psychic aura -- this was the old definition

HP:0011162
name: Psychic aura
def: "Aura with affective, mnemonic or composite perceptual phenomena including illusory or composite hallucinatory events." 
comment: Events may appear alone or in combination. Included are feelings of depersonalization. These phenomena have subjective qualities similar to those experienced in life but are recognized by the subject as occurring outside of actual context.

**Comment by pnrobinson** [comment]
Also, we renamed the term "Aura" to "Focal sensory seizure".
Should we revert the name change and create a new term to make things look like the powerpoint ?
@DLewisSmith

**Comment by DLewisSmith** [comment]
Thanks @pnrobinson. It should be a synonym of 
Focal cognitive seizure with deja vu/jamais vu (HP:0032697), other synonym (Familiarity aura)

**Comment by pnrobinson** [comment]
@DLewisSmith  thanks! Ps one last thing -- do we have a definition for Cognitive aura?

**Comment by pnrobinson** [comment]
done

**Comment by DLewisSmith** [comment]
"A purely subjective manifestation of an epileptic seizure pertaining to altered cognition." This may be a bit of placeholder as it does not explain the exclusivity of Familiarity and Illusionary auras but I can't find an explicit definition for these in the published classification.
